build

```diff
@@ -81,6 +81,11 @@ esac
 set -o xtrace
 set -o errexit
 
+# Check for dependencies
+for dep in docker make mmv; do
+    which ${dep} || { echo "The command '${dep}' is required."; exit 1; }
```

Comment:
nit - `command -v` is a builtin and thus should be "faster"

Reply:
Interesting, didn't know about that one!